bson/raw_bson.py

```diff
@@ -156,7 +160,9 @@ def __inflated(self) -> Mapping[str, Any]:
         return self.__inflated_doc
 
     @staticmethod
-    def _inflate_bson(bson_bytes: bytes, codec_options: CodecOptions) -> Mapping[Any, Any]:
+    def _inflate_bson(
+        bson_bytes: bytes, codec_options: CodecOptions[RawBSONDocument]
+    ) -> Mapping[str, Any]:
```

Comment:
Should return `MutableMapping[str, Any]` to match the signature of `_inflate_bson`.

Reply:
Actually I think it should be `Mapping` everywhere in this file, updated.

Reply:
Actually, this method is somewhat out of place, I think it really should have been in `bson/__init__.py`.  I think it is fine for this to return `MutableMapping` since it is used in other contexts, while the APIs of `RawBSONDocument` return `Mapping`, since it is meant to be a read-only object.